build/repo.targets

```diff
@@ -15,7 +15,7 @@
     <GeneratedPackageVersionPropsPath>$(IntermediateDir)dependencies.props</GeneratedPackageVersionPropsPath>
     <GeneratedRestoreSourcesPropsPath>$(IntermediateDir)sources.props</GeneratedRestoreSourcesPropsPath>
 
-    <PrepareDependsOn>$(PrepareDependsOn);VerifyPackageArtifactConfig;PrepareOutputPaths</PrepareDependsOn>
+    <PrepareDependsOn>InstallDotNet;$(PrepareDependsOn);VerifyPackageArtifactConfig;PrepareOutputPaths</PrepareDependsOn>
```

Comment:
Can we put this into RestoreDependsOn instead? Conceptually, Restore is the "download" stuff step.

Reply:
Sure